.github/workflows/check_pr.yml

```diff
@@ -62,6 +64,17 @@ jobs:
       - uses: actions/checkout@v2
       - uses: dart-lang/setup-dart@v1.3
 
+      - name: Install sudo package
+        run: apt update && apt install sudo
+
+      - name: Start Solana Local Validator
+        uses: switchboard-xyz/solana-local-validator@v0.1
+        with:
+          solana-version: stable
+          cluster: mainnet
+          args:
+            --bpf-program ${{ env.EC_LINKS_ID }} ${{ env.EC_LINKS_SO }}
```

Comment:
According to the action docs, you also need to run it, does it work without it?

Reply:
Yes the local validator should already be running after this action